roles/pulibrary.postgresql/tasks/create_db.yml

```diff
@@ -3,7 +3,7 @@
   postgresql_db:
     name: '{{ application_db_name }}'
     login_host: '{{ postgres_host | default(omit) }}'
-    login_port: '{{ postgres_port | default(omit) }}'
+    port: '{{ postgres_port | default(omit) }}'
```

Comment:
@kayiwa this change was really strange, but I got a no attribute error when running the playbook.

Reply:
surprised it worked given this. https://docs.ansible.com/ansible/2.3/postgresql_db_module.html (there's no `login_port`) Good catch.